new-lamassu-admin/src/pages/Triggers/helper.js

```diff
@@ -223,18 +223,48 @@ const typeSchema = Yup.object()
     'are-fields-set',
     'All fields must be set.',
     ({ triggerType, threshold }, context) => {
+      const errorMessages = {
+        txAmount: () => 'Transaction amount must be a positive number.',
+        txVolume: threshold => {
+          if (
+            (!threshold.threshold || threshold.threshold <= 0) &&
+            (!threshold.thresholdDays || threshold.thresholdDays <= 0)
+          )
+            return 'Transaction volume and number of days must be positive numbers.'
+          else if (!threshold.threshold || threshold.threshold <= 0)
+            return 'Transaction volume must be a positive number.'
+          else if (!threshold.thresholdDays || threshold.thresholdDays <= 0)
+            return 'Number of days must be a positive number.'
+        },
+        txVelocity: threshold => {
+          if (
+            (!threshold.threshold || threshold.threshold <= 0) &&
+            (!threshold.thresholdDays || threshold.thresholdDays <= 0)
+          )
+            return 'Transaction volume and number of days must be positive numbers.'
+          else if (!threshold.threshold || threshold.threshold <= 0)
+            return 'Transaction volume must be a positive number.'
+          else if (!threshold.thresholdDays || threshold.thresholdDays <= 0)
+            return 'Number of days must be a positive number.'
+        },
+        consecutiveDays: () => 'Number of days must be a positive number.'
+      }
+
       const validator = {
-        txAmount: threshold => threshold.threshold >= 0,
+        txAmount: threshold => threshold.threshold > 0,
         txVolume: threshold =>
-          threshold.threshold >= 0 && threshold.thresholdDays >= 0,
+          threshold.threshold >= 0 && threshold.thresholdDays > 0,
         txVelocity: threshold =>
-          threshold.threshold >= 0 && threshold.thresholdDays >= 0,
-        consecutiveDays: threshold => threshold.thresholdDays >= 0
+          threshold.threshold > 0 && threshold.thresholdDays > 0,
+        consecutiveDays: threshold => threshold.thresholdDays > 0
       }
```

Comment:
I was thinking about doing similar custom messages for the validation when editing triggers, but to be consistent with the rest of the code I validated each field individually and used the default yup messages. You can see that here https://github.com/lamassu/lamassu-server/pull/657 and use this for reference https://github.com/jquense/yup/blob/91ace1e8be3fc23c775ec8117c47b406bf29da4a/src/locale.ts. So maybe we should go towards the same approach. Can you ask Rafael about this? he will know better.